qualtrix/api.py

```diff
@@ -46,15 +52,53 @@ async def get_response(request: ResponseModel):
 
 
 @router.post("/redirect")
-async def get_redirect(request: RedirectModel):
+async def intake_redirect(request: RedirectModel):
```

Comment:
I really doubt we will ever go back to the 1.0 method for any reason. I feel safe moving this entirely, and there is no real "retry" scheme i can think of that would benefit the current implementation. It is a single operation with a number of side effects. I am planning to refactor this to be more atomic and put those susbsequent steps not directly related to the workflow on the client side.